Cargo.toml

```diff
@@ -47,18 +47,18 @@ digest = "0.10"
 directories = "5"
 displaydoc = "0.2"
 ethereum-types = "0.14"
-fil_actor_account_state = { version = "15.2" }
-fil_actor_cron_state = { version = "15.2" }
-fil_actor_datacap_state = { version = "15.2" }
-fil_actor_init_state = { version = "15.2" }
-fil_actor_interface = { version = "15.2" }
-fil_actor_market_state = { version = "15.2" }
-fil_actor_miner_state = { version = "15.2" }
-fil_actor_power_state = { version = "15.2" }
-fil_actor_reward_state = { version = "15.2" }
-fil_actor_system_state = { version = "15.2" }
-fil_actor_verifreg_state = { version = "15.2" }
-fil_actors_shared = { version = "15.2", features = ["json"] }
+fil_actor_account_state = { git = "https://github.com/ChainSafe/fil-actor-states", branch = "no-cids-in-here" }
```

Comment:
use published versions

Reply:
As mentioned in the description, this is blocked by https://github.com/ChainSafe/fil-actor-states/pull/320